lib/topic_view.rb

```diff
@@ -208,7 +209,13 @@ def summary(opts = {})
 
   def read_time
     return nil if @post_number > 1 # only show for topic URLs
-    (@topic.word_count / SiteSetting.read_time_word_count).floor if @topic.word_count
+
+    if @topic.word_count && SiteSetting.read_time_word_count > 0
```

Comment:
If you have a topic with 3 posts: OP with some text and 2 replies with only some images.

How does this take into account the replies into the reading time?

Reply:
It depends on the post itself and on the `SiteSetting.read_time_word_count` value.

If the post is short, then the result will be 1 minute of reading (4 seconds per post). If the post is long (> SiteSetting.read_time_word_count words), then the result will be dominated by that value (`word_count / SiteSetting.read_time_word_count` minutes).

Reply:
Nevermind, I thought posts with only an image would have a `word_count == 0` but it turns out they have some words (ie, chunks from the URL).

eg. a post with this `raw` content

``` text
![image|277x128](upload://lYlShwE97fhTvChFLbM3zXyUPnj.png) 
``` 

will have the following "words": image, 277x128, upload, lYlShwE97fhTvChFLbM3zXyUPnj and	png.